blueprints/app/files/app/deprecation-workflow.ts

```diff
@@ -0,0 +1,25 @@
+import setupDeprecationWorkflow from 'ember-cli-deprecation-workflow';
+
+/**
+ * Docs: https://github.com/ember-cli/ember-cli-deprecation-workflow
+ */
+setupDeprecationWorkflow({
+/**
```

Comment:
Personally, I don't like all these comments (we already link to the docs), but I'll leave that to the rest of the team.